docs/development/styles.md

```diff
@@ -0,0 +1,58 @@
+# Styles
+
+## Recommended Components
+
+**TODO**
+
+## Guidance
+
+**TODO**
+
+### Frameworks
+
+* Plain CSS?
```

Comment:
could split up to 3 categories: css, behavior, style system

css: sass, tailwind, less
style systems can be: tailwindui, bootstrap
behavior: headless ui, mui, radix

Reply:
style systems can existing design methodologies

Reply:
javascript frameworks can have their own flavor of the above

Reply:
sass nodejs library: https://github.com/sass/dart-sass, previous node-sass is deprecated

Reply:
for plain css, could mention methodologies of organizing css/sass
```
https://getbem.com/introduction/
[OOCSS](http://oocss.org/) — Separating container and content with CSS “objects”
[SMACSS](http://smacss.com/) — Style-guide to write your CSS with five categories for CSS rules
[SUITCSS](https://suitcss.github.io/) — Structured class names and meaningful hyphens
[Atomic](https://acss.io/) — Breaking down styles into atomic, or indivisible, pieces
```

Reply:
For design systems, we should definitely add Carbon, not just because it's an awesome style system but because its IBM's own creation, and we have a lot of experience with it.